clusters/build-clusters/01_cluster/loki/loki.yaml

```diff
@@ -276,32 +283,6 @@ spec:
     rollingUpdate:
       maxUnavailable: 33%
 ---
-apiVersion: policy/v1beta1
-kind: PodSecurityPolicy
-metadata:
-  name: loki-promtail
-spec:
-  allowPrivilegeEscalation: false
-  fsGroup:
-    rule: RunAsAny
-  hostIPC: false
-  hostNetwork: false
-  hostPID: false
-  privileged: false
-  readOnlyRootFilesystem: true
-  requiredDropCapabilities:
-  - ALL
-  runAsUser:
-    rule: RunAsAny
-  seLinux:
-    rule: RunAsAny
-  supplementalGroups:
-    rule: RunAsAny
-  volumes:
-  - secret
-  - configMap
-  - hostPath
----
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
```

Comment:
You can drop the PSP in this role IMHO. And you need to lift it IIRC to a ClusterRole to make use of SCC privileged

Reply:
> You can drop the PSP in this role IMHO

Done. I will delete those after merge (on build01 and build02).

```
oc --context build01 get role loki-promtail -n loki
oc --context build01 get rolebinding loki-promtail -n loki
```

> And you need to lift it IIRC to a ClusterRole to make use of SCC privileged

Is it enough?

https://github.com/openshift/release/blob/1d0f67ff7dbeb3cbe90dfacda6faea0564a36b16/clusters/build-clusters/01_cluster/loki/loki.yaml#L32-L52